src/interceptors/Socket/MockSocket.ts

```diff
@@ -35,6 +35,19 @@ export class MockSocket extends net.Socket {
     return this
   }
 
+  public address(): {} | net.AddressInfo {
+    // @ts-ignore
+    if (this.responseType === 'mock') {
+      return {
+        address: '::1', // should be this.connectionOptions.host?
+        family: 'IPv4',
+        port: 1,
+      }
+    }
+
+    return super.address()
```

Comment:
This will have incorrect (empty?) value for bypass scenarios because `MockHttpSocket` has no connection information. 

I proposed to infer this from the original socket but now I wonder, `socket.address()` is generally available throughout the socket's life-cycle, yes? Like, you can grab the address at any point in time after constructing the socket. Hmm.